common/include/pcl/point_cloud.h

```diff
@@ -547,6 +646,16 @@ namespace pcl
         height = 1;
       }
 
+      /** \brief Insert a new point in the cloud, at the end of the container.
+        * \note This assumes the user would correct the details later on!
```

Comment:
It might be unclear what "details" means. The formulation "This assumes the user would correct the width and height later on" or similar might be better (more blow)

Reply:
Just replaced `the details` with `the width and height`. Hope it's more clear now :)